src/index.html

```diff
@@ -7,7 +8,52 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Stars</h1>
+    <section class="section">
+      <div class="stars stars--0">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+      <div class="stars stars--1">
+        <span class="stars__star active"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+      <div class="stars stars--2">
+        <span class="stars__star active"></span>
+        <span class="stars__star active"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+      <div class="stars stars--3">
+        <span class="stars__star active"></span>
+        <span class="stars__star active"></span>
+        <span class="stars__star active"></span>
+        <span class="stars__star"></span>
```

Comment:
```suggestion
      <div class="stars stars--3">
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
```
Stars elements shouldn't have any additional classes. Use `stars--N` modifier to set the active stars